doc/source/examples/technology_showcase_examples/techdemo-28/ex_28-tecfricstir.rst

```diff
@@ -1154,11 +1149,14 @@ The observed temperature rise in the model shows that heat generation during
 the second and third load steps is due to friction between the tool shoulder
 and workpiece, as well as plastic deformation of the workpiece material.
 
-The melting temperature of 304L stainless steel is 1450 °C. As shown in the
+.. vale off
+
+The melting temperature of 304L stainless steel is 1450 degrees C. As shown in the
 following figure, the maximum temperature range at the weld line region on the
 workpiece beneath the tool is well below the melting temperature of the
-workpiece material during the second and third load steps, but above 70 percent
+workpiece material during the second and third load steps but above 70 percent
 of the melting temperature:
+.. vale on
```

Comment:
Because of 304L most probably. Vale detects that you are using "units" though they are actually not units, its just a way of naming the steel alloy.. I believe that's why vale was disregarded in this section.

Reply:
If they were units, vale forces you to write them as ``304 L``